package.json

```diff
@@ -3,13 +3,23 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "apollo-cache-inmemory": "^1.1.11",
+    "apollo-cache-persist": "^0.1.1",
+    "apollo-client": "^2.2.7",
+    "apollo-link": "^1.2.1",
+    "apollo-link-context": "^1.0.7",
+    "apollo-link-http": "^1.5.3",
+    "apollo-link-state": "^0.4.1",
```

Comment:
Looks like this should wait until #25 is merged..?

Reply:
It's a bit difficult to review this before #25 is merged, because this is now equal to #25 plus a bit more.